pkg/downloader/chart_downloader.go

```diff
@@ -158,6 +159,66 @@ func (c *ChartDownloader) DownloadTo(ref, version, dest string) (string, *proven
 	return destfile, ver, nil
 }
 
+type inMemoryChart struct {
+	ChartData              *bytes.Buffer
+	Version                string
+	ProvenanceVerification *provenance.Verification
+}
+
+type InMemoryChartList map[string]inMemoryChart
```

Comment:
Please remove.  I don't want to expose any detail about the cache.  Please see comment on `GetInMemoryCache()` for an explanation

Reply:
Had to add the new type because of the go-lint, but have made it private now